extensions/netlify/package.json

```diff
@@ -43,10 +44,10 @@
       "mode": "view"
     },
     {
-      "name": "search-domains",
-      "title": "Search Domains",
+      "name": "search-domains-and-manage-dns-records",
```

Comment:
Can we keep the name since hotkeys, aliases and ranking are using the name as ID 🙂

Reply:
Hi.
I'm not really sure what you mean but yeah we can revert the name change if required. Lemme quickly make the change.

Reply:
So the name is the ID for the command, if you change that then the users will lose the ranking / alias / hotkey attached 🙂

Thanks for changing back 🙂